lib/rsa/rsa-verify.c

```diff
@@ -291,7 +291,7 @@ static int rsa_verify_key(struct image_sign_info *info,
 			  const uint32_t key_len)
 {
 	int ret;
-#if !defined(USE_HOSTCC)
+#if !defined(USE_HOSTCC) && !(defined(CONFIG_SPL_RSA) && defined(CONFIG_IMX8MM))
```

Comment:
Can you explain why we need these changes?

I would also prefer an option that is not bound to imx8, as this is actually generic stuff (and it surprises me this is not an issue with imx6/7).

Reply:
the commit and the target of this work is imx8mm so this addresses imx8mm leaving other platforms unaffected
but sure, when we add support for other boards, this will have to be modified and a more generic definition done.

Reply:
But why isn't this an issue with imx6/imx7?

Reply:
we must  be talking past each other :)  imx6 and imx7 don't define (CONFIG_SPL_RSA && CONFIG_IMX8MM) so this leaves the previous implementation the way it was.

if you are saying the previous implementation is broken, sure lets fix it on a separate commit by extending the changes done in this one.

or if you are saying we want imx6 and imx7 to now work the same way that imx8mm will work after this commit sure, lets change it...but IMO should be on a separate commit, not pushing it all together...

Reply:
What I'm asking is why do we need to change the generic rsa-verify function only for IMX8MM if this code path seems to be shared with imx6/imx7.

I didn't get why we need this change from your commit, it is not clear.

Reply:
ah right, it is because those platforms somehow manage to define USE_HOSTCC and I wasnt able to enable such definition